v4/iacr.css

```diff
@@ -84,22 +85,22 @@
 
 
 /*** Search ***/
-img.searchBtn {
+img.searchIcon {
   height: 20px;
 }
 
 .searchButtons {
   background-color: transparent;
-  border-color: #ffffff;
-  color: #ffffff;
+  color: #F0F0F0;
```

Comment:
I assume this is to make things consistent in the menu?

Reply:
Yes